src/main/jte/admin/opening-hours.jte

```diff
@@ -51,7 +51,8 @@
                 </tbody>
             </table>
             <button type="submit"
-                    class="btn btn-primary">${context.localize("administration.opening_hours.save")}</button>
+                    class="btn btn-primary" aria-label="${context.localize("administration.opening_hours.save")}"
+                    title="${context.localize("administration.opening_hours.save")}">${context.localize("administration.opening_hours.save")}</button>
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.
Also, this should probably be converted into an `<input "type=submit"`